lib/tds.js

```diff
@@ -527,8 +571,7 @@ async function generateTdsRuleset (
             dnrRules.push(rule)
         }
     }
-
-    return finalizeDNRRulesAndLookup(startingRuleId, dnrRules)
+    return { ...finalizeDNRRulesAndLookup(startingRuleId, dnrRules), customActionRules }
```

Comment:
I took a look though I'm not clear where the `supported-surrogates-input.json` is supposed to come from?    It doesn't work with `mapping.json` since it doesn't parse as a Set?

Reply:
Sorry, I need to update the README about that. For testing, I have a `supported-surrogates.json` file that contains `["analytics.js"]`.